src/utils/shared-default-props.js

```diff
@@ -31,6 +31,7 @@ const PickerDefaultProps = {
   emojisToShowFilter: null,
   showPreview: true,
   showSkinTones: true,
+  darkMode: false,
```

Comment:
The default should probably be based on prefers-color-scheme: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme